tfe/resource_tfe_variable.go

```diff
@@ -276,7 +276,7 @@ func resourceTFEVariableSetVariableRead(d *schema.ResourceData, meta interface{}
 	// Update config.
 	d.Set("key", variable.Key)
 	d.Set("category", string(variable.Category))
-	d.Set("description", string(variable.Description))
+	d.Set("description", variable.Description)
```

Comment:
removes redundant string conversion, since `Description` is of type `string` in the struct